test/corpus/misc.txt

```diff
@@ -136,7 +136,7 @@ effective date
 ================================================================================
 
 2020-12-02=2020-12-03
-    Expenses:Food:Groceries    ; [=2020-12-04]
```

Comment:
Is this a mistake/temporary local edit that you forgot to revert?

Reply:
No, not a mistake. I moved that to the `comments_and_notes.txt` tests (title: "with effective date") because it seemed to fit in there. I recall that I broke it while working on my changes to notes, but I was only running the `comments_and_notes` tests so didn't notice until after. But now I'm questioning my judgement on moving it. It's certainly not important (or related to this change!) and I can revert it if you'd prefer.